Resources/config/controller.xml

```diff
@@ -17,6 +17,7 @@
             <argument type="service" id="liip_monitor.helper.runner_manager" />
             <argument type="service" id="liip_monitor.helper" />
             <argument>%liip_monitor.view_template%</argument>
+            <argument type="service" id="logger" on-invalid="ignore" />
```

Comment:
It would be fine as mandatory - not having a logger service is a little silly anyhow.

I have tested the symfony-demo app (v2.7.6) with all monolog configs removed, and you still get a valid `LoggerInterface` (a `Symfony\Bridge\Monolog\Logger`) - but it has no handlers and so will just try to send to STDERR.